livingatlas/pipelines/src/main/java/au/org/ala/kvs/cache/CountryKeyValueStore.java

```diff
@@ -20,7 +19,9 @@ public static CountryKeyValueStore create(GeocodeShpConfig config) {
   }
 
   @Override
-  public void close() throws IOException {}
+  public void close() {
+    // NOP
```

Comment:
What does NOP mean ?

Reply:
No operation, usually an assembly language instruction to do nothing.

https://en.wikipedia.org/wiki/NOP_(code)